java/arcs/core/util/Scheduler.kt

```diff
@@ -237,6 +238,8 @@ class Scheduler(
             return copy(listeners = this.listeners + (listener.namespace to listeners))
         }
 
+        fun clear() { listeners = emptyMap() }
```

Comment:
can this be fun clear() { listeners.clear() }
then you can keep it as a val

Reply:
Map doesn't have a clear() api.
I thought about using MutableMap instead though It needs a bit more modifications.

Hmm, but I think keeping val would be good practice.
Let me change it.

Reply:
done